cmd/manager/main.go

```diff
@@ -225,3 +239,50 @@ func checkIfRequiredFilesExist(requiredFiles []string) error {
 	}
 	return nil
 }
+
+// clusterConfigChecks checks for required configuration in a cluster before starting the controller.
+// If the cluster doesn`t meet the configuration, it returns an error.
+func clusterConfigChecks(cfg *rest.Config) error {
+	// Get the OpenShift API config client.
+	clienset, err := configclient.NewForConfig(cfg)
+	if err != nil {
+		return errors.Wrap(err, "error creating the OpenShift API config client")
+	}
+
+	// Get the OpenShift operator client
+	operatorClient, err := operatorv1.NewForConfig(cfg)
+	if err != nil {
+		return errors.Wrap(err, "error creating the OpenShift API operator client")
+	}
+	// Check for valid K8s version. Currently any version >= 1.18 is supported.
+	// The OpenShift version is not explicitly used to acquire binaries. Hence the check for OpenShift version is
+	// not required.
```

Comment:
You need to mention why. The reason being the CI clusters returns version in `0.0.x` format or whatever that format is.